src/main/java/seedu/duke/constants/ErrorMessage.java

```diff
@@ -8,26 +8,27 @@ public interface ErrorMessage {
     String INVALID_AMOUNT_TO_ADD = "Please provide a value greater than 0";
     String INVALID_CURRENCY = "An invalid currency has been provided.";
     String INVALID_NUMERICAL_AMOUNT = "Please provide a float amount to update";
-    String INVALID_CREATE_ACCOUNT_COMMAND = "Please check that you have correctly provided " +
-            "the currency of the account to add";
+    String INVALID_CREATE_ACCOUNT_COMMAND = "create-account must be used with the following format: " +
+            "create-account <CURRENCY>";
```

Comment:
```suggestion
            "create-account CURRENCY";
```
I think we should follow the user guide syntax for this